wow/core/src/main/scala/wow/realm/entities/CharacterRef.scala

```diff
@@ -0,0 +1,25 @@
+package wow.realm.entities
+
+import scodec.bits.ByteVector
+
+/**
+  * Created by sknz on 5/19/17.
+  */
```

Comment:
Could you be a bit more specific? I understand the people like us can deduce by cross examining the commits details made on the 19th of May and it's hashes the purpose of this object, but the common mortal contributor such as JM and others do not have the same skills